openmetadata-ui/src/main/resources/ui/src/components/DataQuality/TestSuite/TestSuiteList/TestSuites.component.tsx

```diff
@@ -276,88 +299,91 @@ export const TestSuites = () => {
-              testSuitePermission?.Create && (
-                <Link
-                  data-testid="add-test-suite-btn"
-                  to={ROUTES.ADD_TEST_SUITES}>
-                  <Button type="primary">
-                    {t('label.add-entity', { entity: t('label.test-suite') })}
-                  </Button>
-                </Link>
-              )}
-          </Col>
-        </Row>
+        <Form layout="inline">
+          <Space align="center" className="w-full justify-between" size={16}>
+            <Form.Item className="m-0" label={t('label.owner')} name="owner">
+              <UserTeamSelectableList
+                hasPermission
+                owner={selectedOwner}
+                onUpdate={(updatedUser) => handleOwnerSelect(updatedUser)}>
+                <Select
+                  data-testid="owner-select-filter"
+                  open={false}
+                  placeholder={t('label.owner')}
+                  value={ownerFilterValue}
+                />
+              </UserTeamSelectableList>
+            </Form.Item>
+          </Space>
+        </Form>
       </Col>
 
       <Col span={24}>
-        <SummaryPanel
-          showAdditionalSummary
+        <PieChartSummaryPanel
           isLoading={isTestCaseSummaryLoading}
           testSummary={testCaseSummary}
         />
       </Col>
+
       <Col span={24}>
-        <Table
-          columns={columns}
-          customPaginationProps={{
-            currentPage,
-            isLoading,
-            pageSize,
-            isNumberBased: true,
-            paging,
-            pagingHandler: handleTestSuitesPageChange,
-            onShowSizeChange: handlePageSizeChange,
-            showPagination,
-          }}
-          data-testid="test-suite-table"
-          dataSource={testSuites}
-          loading={isLoading}
-          locale={{
-            emptyText: <FilterTablePlaceHolder />,
-          }}
-          pagination={false}
-          scroll={{
-            x: '100%',
-          }}
-          size="small"
-        />
+        <div className="test-suite-list-container">
+          <div className="test-suite-list-header">
+            <Row gutter={[16, 16]}>
+              <Col data-testid="test-suite-sub-tab-container" span={16}>
+                <Radio.Group value={subTab} onChange={handleSubTabChange}>
+                  <Radio.Button
+                    data-testid="table-suite-radio-btn"
+                    value={DataQualitySubTabs.TABLE_SUITES}>
+                    {t('label.table-suite-plural')}
+                  </Radio.Button>
+                  <Radio.Button
+                    data-testid="bundle-suite-radio-btn"
+                    value={DataQualitySubTabs.BUNDLE_SUITES}>
+                    {t('label.bundle-suite-plural')}
+                  </Radio.Button>
+                </Radio.Group>
+              </Col>
+              <Col span={8}>
+                <Searchbar
+                  removeMargin
```

Comment:
Out table component gives you ability to pass this things to render it in header, have you consider that?

Reply:
Yes i checked that, but it has fixed left side search bar and extra prop at right side, and as per design we need this radio button as at left side. and search at right side.